src/gui/owncloudgui.cpp

```diff
@@ -347,6 +348,15 @@ void ownCloudGui::addAccountContextMenu(AccountStatePtr accountState, QMenu *men
     actionOpenoC->setProperty(propertyAccountC, QVariant::fromValue(accountState->account()));
     QObject::connect(actionOpenoC, SIGNAL(triggered(bool)), SLOT(slotOpenOwnCloud()));
 
+    if (accountState->state() == AccountState::Connected) {
+        menu->addSeparator();
+
+        QString quotaInfo = tr("Fetching Quota");
+        auto actionQuotaInfo = menu->addAction(quotaInfo);
+        actionQuotaInfo->setEnabled(false);
+        new QuotaAction(actionQuotaInfo, accountState);
```

Comment:
The hacks for the tray on OS X you mean? Because I checked and on linux/windows the requests are just every 30 seconds.